test/Faker/Container/ContainerBuilderTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Faker\Test\Container;
+
+use Faker\Container\ContainerBuilder;
+use Faker\English\FileExtension;
+use Faker\Test\TestCase;
+use Psr\Container\ContainerInterface;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+final class ContainerBuilderTest extends TestCase
+{
+    public function testBuildEmpty()
+    {
+        $builder = new ContainerBuilder();
+        $container = $builder->build();
+
+        self::assertInstanceOf(ContainerInterface::class, $container);
+    }
+
+    public function testBuild()
+    {
+        $builder = new ContainerBuilder();
+        $builder->add(FileExtension::class);
+        $container = $builder->build();
+
+        self::assertInstanceOf(ContainerInterface::class, $container);
+    }
+
+    public function testBuildWithDuplicates()
+    {
+        $builder = new ContainerBuilder();
+        $builder->add(FileExtension::class);
+        $builder->add(FileExtension::class);
+        $container = $builder->build();
+
+        self::assertInstanceOf(ContainerInterface::class, $container);
+    }
+
+    public function testBuildWithObject()
+    {
+        $builder = new ContainerBuilder();
+        $builder->add(new FileExtension(), 'foo');
+        $container = $builder->build();
+
+        self::assertInstanceOf(ContainerInterface::class, $container);
```

Comment:
What are we testing here?

All of these tests would pass if `$builder->build()` returns an empty container.

Reply:
This test make sure that the logic to parse input is valid. 

https://github.com/FakerPHP/Faker/pull/154/files#diff-e901ca85e7a858f528ed791969de6ee00de3cd11eae9da9822e44a994ace72bdR30-R32